lms/djangoapps/courseware/tabs.py

```diff
@@ -66,8 +68,7 @@ def is_enabled(cls, course, user=None):
         """
         The "Home" tab is not shown for the new unified course experience.
         """
-        request = RequestCache.get_current_request()
-        return not waffle.flag_is_active(request, UNIFIED_COURSE_EXPERIENCE_FLAG)
+        return not course_experience_config().is_enabled(UNIFIED_COURSE_EXPERIENCE_FLAG, course.id)
```

Comment:
This is close to what I was thinking. Here's what I had in mind:

    return COURSE_EXPERIENCE_FEATURE_FLAG.is_enabled(course.id)

i.e. that there would be a constant which was an instance of the feature flag, and client code could use it without knowing anything more. This allows the client to only import one thing (the feature flag variable) instead of always having to import two (the config method and the flag name).

IMO it is best not to introduce a new concept called a config at this point, unless we think that it will handle all course experience configuration.